tools/minimerge/import.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e
+
+DIR="$(realpath "$(dirname $0)")"
+REPO_DIR="$(realpath "${DIR}/../../")"
+TOOL="${DIR}/minimerge_tool"
+METADATA_FILE="${DIR}/cuttlefish_import_metadata.txt"
+CUTTLEFISH_REPO_URL="https://android.googlesource.com/device/google/cuttlefish"
+
+echo "Importing AOSP's device/google/cuttlefish project"
+
+# Do these early to avoid doing unnecessary work when these fail
+# Find last imported commit
+stat "${METADATA_FILE}" >/dev/null
+last_imported_commit="$(grep '^[[:space:]]*last_import[[:space:]]*:' "${METADATA_FILE}" | cut -d: -f2 | grep -o -E '[^[:space:]]+' )"
+# Find current commit
+pushd "${DIR}" >/dev/null
+[[ `git status --porcelain` ]] >/dev/null && (echo "Repository is not in clean state"; exit)
```

Comment:
Is this trying to test if `git status --porcelain` returns non-empty output? On my machine `git status --porcelain` writes to `stderr` so I'm not how that is supposed to work.

It seems to work as intended in `bash`, but in `zsh` I need to use
```bash
[[ `git status --porcelain 2>&1` ]]
```
for this to work consistently. Not sure how important it is since the file explicitly uses `/bin/bash`.

Also `[[` is not POSIX, it's a [bash extension](https://stackoverflow.com/a/47576482).

Reply:
TBH I found this on stackoverflow.

It's ok if it only works on bash.